tasks/docker.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Add Docker APT key
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+
+- name: Install required packages for Docker
```

Comment:
Can't you install all packages in one task, after the repository has been added?